JS-Algorithms/challenges.js

```diff
@@ -70,7 +103,13 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  let arrayNumber=[];
+  for (let i = 1; i <=number; i++) {
+    arrayNumber.push(BigInt(i)**BigInt(i));
+  }
+  let result=arrayNumber.reduce((acc, curr) => acc + curr, 0n);
```

Comment:
Use always `const`, if you need to reassign a value or change the reference, you can change it to `let` but I would recommend avoiding `let` for most of the cases because can cause accidental bugs.

```suggestion
  const arrayNumber=[];
  for (let i = 1; i <=number; i++) {
    arrayNumber.push(BigInt(i)**BigInt(i));
  }
  const result=arrayNumber.reduce((acc, curr) => acc + curr, 0n);
```